x-pack/plugins/spaces/public/management/components/confirm_delete_modal/confirm_delete_modal.tsx

```diff
@@ -5,224 +5,127 @@
  * 2.0.
  */
 
-import './confirm_delete_modal.scss';
-
-import type { CommonProps, EuiModalProps } from '@elastic/eui';
-import {
-  EuiButton,
-  EuiButtonEmpty,
-  EuiCallOut,
-  EuiFieldText,
-  EuiFormRow,
-  EuiModal,
-  EuiModalBody,
-  EuiModalFooter,
-  EuiModalHeader,
-  EuiModalHeaderTitle,
-  EuiSpacer,
-  EuiText,
-} from '@elastic/eui';
-import type { ChangeEvent } from 'react';
-import React, { Component } from 'react';
-
-import type { InjectedIntl } from '@kbn/i18n/react';
-import { FormattedMessage, injectI18n } from '@kbn/i18n/react';
+import { EuiCallOut, EuiConfirmModal, EuiSpacer, EuiText } from '@elastic/eui';
```

Comment:
Using the base `EuiConfirmModal` here makes me so happy. Thanks for taking the time to clean this up! It had a great side-effect of reducing our async chunk size too:

![image](https://user-images.githubusercontent.com/3493255/118171645-7371d280-b3f9-11eb-911e-b34ee9514010.png)